src/SourceTerm/Manager.cpp

```diff
@@ -123,11 +123,22 @@ void seissol::sourceterm::transformNRFSourceToInternalSource( glm::dvec3 const&
   faultBasis[6] = subfault.normal.x;
   faultBasis[7] = subfault.normal.y;
   faultBasis[8] = subfault.normal.z;
-
-  double mu = (subfault.mu == 0.0) ? material.mu : subfault.mu;
-  pointSources.muA[index] = mu * subfault.area;
-  pointSources.lambdaA[index] = material.lambda * subfault.area;
-
+  
+  pointSources.A[index] = subfault.area;
+  switch(material->getMaterialType()) {
+    case seissol::model::MaterialType::anisotropic:
+      if (subfault.mu != 0) {
+        logWarning(0) << "There are specific fault parameters for the fault. This version of SeisSol was compiled for anisotropic materials. This is only compatible if the material around the source is actually isotropic.";
+      }
```

Comment:
As before, should we fail directly with an error in such situations?

Reply:
Yes, better make it an error.